### DISCUSSION TITLE
Welcome to Memoria Discussions!

### DISCUSSION THREAD

**Discussion by Albeoris** [Albeoris/Memoria#166] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by barkermn01** [comment]
Is there a Discord server for this project? I ask because I'm one of the programmers that work on 7th heaven the Final Fantasy 7 Modding platform, and it would be good to have a chat with some devs about what someone has asked me to do?

**Reply by Tirlititi** [reply]
I am on Qhimm's discord. Albeoris and I are on Steam.

**Comment by Fxcloud90** [comment]
Is it possible for Vivi's attack to use magic while using auto attack?

**Reply by snouz** [reply]
There's already a function for that in memoria

In memoria.ini, under [Battle]
ViviAutoAttack = 1
(0 = Default attack / 1 = One basic elemental spell (cost-free MP))

**Comment by FullFollyFraust** [comment]
Hello my name is Casey. I'm very new to all this. I have installed Memoria, Moguri, Hades workshop tool and I was wondering if it's possible to edit model textures. For example editing Amarant's shoulder tattoos. And replacing the boss battle theme with a different song. Any help would be awesome. I've found where to find the model textures in Hades Workshop Tool Unity Assets Viewer p0data4 but where to go from there I have no idea. Previously I've had a blast editiing 2D sprites in GIMP for Darkest Dungeon.

**Reply by Tirlititi** [reply]
Hello.
You're almost done: once you have identified the model textures you want to modify, right-click on them in the Unity Assets Viewer and use "Export Selection". This will generate the texture(s) in a folder ``HadesWorkshopAssets/p0data4/...`` in your game's directory, in PNG format by default (which is usually the best option, but you can change that with the menu "Automatically convert textures").
In order to use these extracted textures in a mod, you just need to move them to a different folder. Instead of:
```
HadesWorkshopAssets/p0data4/{texturepath}
```
... you need to move the file to:
```c
MyMod/StreamingAssets/{texturepath}
// so it would be something like "MyMod/StreamingAssets/assets/resources/models/2/98_0.png"
```
The same goes with musics: you need to place your custom music file, in format OGG, with the correct path in your mod's folder.

More infos on the wiki:
[Mod folders and custom assets](https://github.com/Albeoris/Memoria/wiki/Mod-folders)
[Tuto: add music](https://github.com/Albeoris/Memoria/wiki/Add-a-new-music)

**Reply by FullFollyFraust** [reply]
Hey thanks for the help and resource links! Is there a way to search the unity assets viewer quickly with a specific keyword instead of scrolling? I'm excited to try this out I'll let you know if I have more questions or bumps in the road

**Reply by FullFollyFraust** [reply]
While using the Moguri mod redoing model textures. Is making a custom mod through the p0data04 in the base FF9_Launcher going to apply to the Moguri mod version of the texture?

**Reply by FullFollyFraust** [reply]
I'm probably just being dense lol They indeed appear to be the Moguri textures now that I'm looking at them in GIMP

**Reply by FullFollyFraust** [reply]
I haven't been able to get my mod to work yet. I followed and created the file path in your instructions. I'm not sure though if I have done the Memoria.ini mod registering part correctly. The mod is appearing in the Memoria launcher and is turned on but isn't showing the changes in game.
* SOLVED *

**Reply by FullFollyFraust** [reply]
Also I wanted to thank you for your work on the playable characters pack! I especially love being able to have Blank all the time and Garnet in her White mage outfit. I did notice she has some hair clipping out in the front hood part of the battle model(I don't know if there is a fix for this)but I still love it! Just wanted to throw some much deserved appreciation your way.

**Comment by WarpedEdge** [comment]
PCP does not add Blank, it just enables all characters at any time. PCP
does add some new characters. The recent update I did added 11 characters
on top of the 3 Tirlititi did. If you wanted a fully fledged Tantalus
Troupe, I recommend the Tantalus mod.

**Reply by FullFollyFraust** [reply]
You're right I have the Tantalus mod installed as well. I misremembered and mis-credited where Blank was from. I've also really been enjoying Puck and Fratley in my trio playthrough with Freya. When I was a kid playing FF9 for the first time I really wanted Puck, Fratley, and Blank to join my team :D

**Comment by jshmo-wav** [comment]
I cannot quite figure out how to add custom fonts that aren't preexisting in the mod? I have downloaded the ttf file to add it but need assistance on the next step in the process if someone can help.

**Reply by Tirlititi** [reply]
Hello.
You need to install the font on your machine for all users (on Windows for example, it's done by right-clicking the TTF file and select something along "Install for all users"). Then it should appear in the font list on the launcher (you may need to launch the game once before it appears, because the fonts are registered on "Play", not on the launcher).

**Reply by jshmo-wav** [reply]
> Hello. You need to install the font on your machine for all users (on Windows for example, it's done by right-clicking the TTF file and select something along "Install for all users"). Then it should appear in the font list on the launcher (you may need to launch the game once before it appears, because the fonts are registered on "Play", not on the launcher).

I right click it in the download folder but it just acts like a regular file. Should I put it somewhere before doing it because I’m genuinely just slow and haven’t seen anyone make a video on it.

**Reply by jshmo-wav** [reply]
> Hello. You need to install the font on your machine for all users (on Windows for example, it's done by right-clicking the TTF file and select something along "Install for all users"). Then it should appear in the font list on the launcher (you may need to launch the game once before it appears, because the fonts are registered on "Play", not on the launcher).

I found where it says install for all users but nothing changed for it, so I am still a little confused

**Comment by WarpedEdge** [comment]
You installed the font for all users? Have you tried restarting your
computer then checking the font options in the mod launcher afterwards?

**Reply by jshmo-wav** [reply]
I did try that and when I open the memoria.ini file, the font is there.  For some reason, when I open the launcher, the font is nowhere to be seen.

**Reply by WarpedEdge** [reply]
Hmm sorry, its supposed to appear after fully launching the game once after installing the font. Could troubleshoot more on discord if you ping me.
